examples/status-code-propagation/src/propagate_status_code.rs

```diff
@@ -45,19 +45,15 @@ impl Plugin for PropagateStatusCode {
                     // update the value if present (second parameter)
                     // insert a value if not (third parameter)
```

Comment:
this comment is probably not correct anymore

Reply:
good catch!

done: 9eb8fb20084e260ddcab822416907048517cf985